log/log.go

```diff
@@ -30,7 +30,7 @@ func InitializeLogger() *zap.Logger {
 
 }
 
-func LogInfo(message string) {
+func GlInfo(message string) {
```

Comment:
`log.Gl.Info` that I told you earlier has a meaning, it means from the `log` package, get `Gl` which is an exported variable, and call its `Info` method.
Reviewing these things will help you get it better:
- Packages in Go
- Exported variables in Go
- Methods in Go

----
remove `GlInfo` function which you defined here.